src/TrkAnaTreeMaker_module.cc

```diff
@@ -196,7 +200,6 @@ namespace mu2e {
 
       // main TTree
       TTree* _trkana;
-      TProfile* _tht; // profile plot of track hit times: just an example
```

Comment:
Just to check: are we sure we want to remove this plot? I never used it but I could imagine it might be useful to have hit time distributions without having to have diagLevel set to 2. Maybe something we can ask trkana users?